packages/calling-component-bindings/src/hooks/TypeAssertions.tsx

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { EmptySelector, getSelector } from './usePropsFor';
+import React from 'react';
+import { VideoGallerySelector } from '../videoGallerySelector';
+import {
+  CameraButton,
+  DevicesButton,
+  EndCallButton,
+  ErrorBar,
+  MicrophoneButton,
+  ParticipantList,
+  ParticipantsButton,
+  ScreenShareButton,
+  VideoGallery
+} from '@internal/react-components';
+import {
+  CameraButtonSelector,
+  DevicesButtonSelector,
+  MicrophoneButtonSelector,
+  ScreenShareButtonSelector
+} from '../callControlSelectors';
+import { ParticipantListSelector } from '../participantListSelector';
+import { ParticipantsButtonSelector } from '../participantsButtonSelector';
+import { ErrorBarSelector } from '../errorBarSelector';
+
+/**
+ * This function is a compile type check that {@link GetSelector} returns
+ * values of the correct type.
+ *
+ * @private
```

Comment:
These type assertions were added before my changes for how `GetSelector` works and they passed through the refactor. So we're confident that the final types are unchanged.

Reply:
how are these functions to be used? is this just for testing?

Reply:
These are internal type assertions. Making sure that we get the expected concrete types in the end.